atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleDdlConfig.java

```diff
@@ -99,4 +99,11 @@ protected final void checkOracleConfig() {
                 "Oracle 'overflowTablePrefix' cannot be more than 6 characters long.");
 
     }
+
+    @Value.Default
+    @Override
+    public int fetchBatchSize() {
+        // Oracle Arrays can be much larger than Postgres select-where-in way of doing get
+        return 1024;
```

Comment:
What client side memory footprint does this require for various queries? Do we have any of the perf benchmark runs with different values?